CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Use the SpatialOS runtime version 14.5.0 by default.
 - Config setting bPreventAutoConnectWithLocator has been renamed to bPreventClientCloudDeploymentAutoConnect. It has been moved to GDK Setting. If using this feature please update enable the setting in GDK Settings.
 - USpatialMetrics::WorkerMetricsRecieved was made static.
+- Added the ability to connect to a local deployment when launching on a device by checking "Connect to a local deployment" and specifying the local IP of your computer in the Launch dropdown.
```

Comment:
This corresponds to these engine PRs: 
https://github.com/improbableio/UnrealEngine/pull/394
https://github.com/improbableio/UnrealEngine/pull/428